automation_tools/__init__.py

```diff
@@ -2320,7 +2278,7 @@ def product_install(distribution, create_vm=False, certificate_url=None,
         execute(setup_bfa_prevention, host=host)
     execute(fix_qdrouterd_listen_to_ipv6, host=host)
 
-    if create_vm and 'base' in target_image:
+    if os.environ.get('PYTHON_CODE_COVERAGE') == 'true':
```

Comment:
Cannot use removed `create_vm` param so I introduced new job param `PYTHON_CODE_COVERAGE` similar to existing `RUBY_CODE_COVERAGE`